src/main/java/com/conveyal/taui/controllers/RegionController.java

```diff
@@ -93,7 +93,7 @@ public static void fetchOsmAndCensusDataInThread (String _id, Map<String, List<F
                     // Download census data
                     region.statusCode = Region.StatusCode.DOWNLOADING_CENSUS;
                     Persistence.regions.put(region); // save the status
-                    region.opportunityDatasets = SeamlessCensusGridExtractor.retrieveAndExtractCensusDataForBounds(region.bounds, region._id);
+                    // TODO FIX region.opportunityDatasets = SeamlessCensusGridExtractor.retrieveAndExtractCensusDataForBounds(region.bounds, region._id);
```

Comment:
This temporarily disables seamless census extracts?

I think we discussed not doing it automatically on region creation, but making it an option on the opportunity datasets page.